Assets/AppCenter/Plugins/iOS/Crashes/CrashesDelegate.mm

```diff
@@ -8,8 +8,11 @@
 
 static bool (*shouldProcessErrorReport)(MSErrorReport *);
 static NSArray<MSErrorAttachmentLog *>* (*getErrorAttachments)(MSErrorReport *);
+static void (*sendingErrorReport)(MSErrorReport *);
+static void (*sentErrorReport)(MSErrorReport *);
+static void (*failedToSendErrorReport)(MSErrorReport *);
 
-static UnityCrashesDelegate *unityCrashesDelegate = NULL;
+static UnityCrashesDelegate *unityCrashesDelegate;
```

Comment:
Redundant variable

Reply:
NVM, I'm sorry, that's for memory management. The SDK keeps only a weak reference. Could you please mention this in the comment?